lightstep/util.py

```diff
@@ -20,9 +20,7 @@ def _generate_guid():
     """
     Construct a guid - random 64 bit integer converted to a string.
     """
-    # Note: uuid.uuid4() returns 128 bit int. To get 64 bit int, apply the mask.
-    guid = uuid.uuid4().int & (1<<64)-1
-    return str(guid)
+    return str(random.getrandbits(64))
```

Comment:
perhaps we should seed the rng (or use `jumpahead`) somewhere... I'm nervous about two processes that don't otherwise use this package getting the same numbers. (I didn't look to see whether `random`'s global RNG is seeded by default)